boards/SOLAR_V4/board.h

```diff
@@ -61,7 +61,7 @@
  * IO pins assignments.
  */
 #define GPIOA_LED                   0U
-#define GPIOA_SHDN_STM              1U
+#define GPIOA_SHDN_LT1618           1U
```

Comment:
I'm now thinking this should actually be GPIOA_LT1618_EN to match the nomenclature of the schematic (the net is labelled LT1618_ENABLE) and to indicate the nature of the action. Asserting this pin high enables the LT1618, even though the pin on the LT1618 is labelled "!SHDN" it basically is an enable.
```suggestion
#define GPIOA_LT1618_EN             1U
```